sdk/enter.go

```diff
@@ -44,8 +44,10 @@ client_id = {{.ClientID}}
 client_secret = {{.ClientSecret}}
 {{else}}{{if eq .Type "service_account"}}
 [Credentials]
-gs_service_key_file = {{.Path}}
+gs_service_key_file = {{.CredsPath}}
```

Comment:
That claims it is for maps, not structs. I think I didn't catch it just because it is inside a conditional that didn't apply to my testing.